spec.html

```diff
@@ -3574,7 +3569,7 @@ <h1>Await</h1>
           1. Perform ! PerformPromiseThen(_promise_, _onFulfilled_, _onRejected_).
           1. Remove _asyncContext_ from the execution context stack and restore the execution context that is at the top of the execution context stack as the running execution context.
           1. Set the code evaluation state of _asyncContext_ such that when evaluation is resumed with a Completion _completion_, the following steps of the algorithm that invoked Await will be performed, with _completion_ available.
-          1. Return.
+          1. Return NormalCompletion(~unused~).
```

Comment:
why does this one need an explicit return statement at all?

Reply:
It's not strictly necessary, but I think it's helpful so that the "this" in the immediately following note is clear.

Reply:
Because the subsequent NOTE-step needs something to refer to?

Reply:
I'm sure the note could be reworded to refer to the implicit return behavior without the need for a placeholder step, if that were desired.

Reply:
> Because the subsequent NOTE-step needs something to refer to?

Yes. It could probably be rephrased, but I like this wording.